datalad/distribution/add_sibling.py

```diff
@@ -99,17 +91,11 @@ class AddSibling(Interface):
                 This option is ignored if there is already a configured sibling
                 dataset under the name given by `name`""",
             constraints=EnsureStr() | EnsureNone()),
-        recursive=recursion_flag,
-        fetch=Parameter(
-            args=("--fetch",),
-            action="store_true",
-            doc="""fetch the sibling after adding"""),
```

Comment:
Nope, I am just slowly disassembling the command for sanity reasons. All functionality remains the same via the `siblings` command.